### DISCUSSION TITLE
[Tests] Validation needed for data fields in Theme

### DISCUSSION THREAD

**Discussion by rudokemper** [Terrastories/terrastories#754] [Temporarily Archived Tickets]
**Describe the bug**
It's possible to add an online Mapbox map for a community Theme in the Administrative back end. This requires both a Mapbox style url and a Mapbox access token. 

Currently, it's possible to set one of these without the other. This will result in the  app not loading anything when going to the map view. 

**To Reproduce**
1. Go to Theme in the administrative back end (with a username with admin privilege)
2. Edit the theme and add a Mapbox style url.
3. Don't add anything in Mapbox access token.
4. Update Theme.
5. Go to map view - it will be blank.

**Expected behavior**
We should make it so that if `Mapbox style url` field has content, then `Mapbox access token` is required to have content as well, and vice versa.

**Screenshots**
![image](https://user-images.githubusercontent.com/31662219/103091934-b4d92100-45c3-11eb-89cd-cc3572d154f7.png)

**Additional context**

**Comment by everly-gif** [comment]
Hi @rudokemper I will like to try and resolve this. Could I work on this issue?

**Comment by rudokemper** [comment]
Sure, I assigned you. Thanks!

**Comment by everly-gif** [comment]
@rudokemper  how can i access the administraive backend? is there a port number?

**Comment by rudokemper** [comment]
When you log in (see seed data for a sample community - think its admin@terrastories.com, pw terrastories) there is a button on the welcome screen to access it.

**Comment by everly-gif** [comment]
Hi right now I'm at the theme page, I was thinking a simple react validation would work for this but I can't seem to find the relevant .jsx file for theme dashboard

**Comment by rudokemper** [comment]
Hi, this too i would ask in the Terrastories Slack channel which I shared in the other thread.

**Comment by everly-gif** [comment]
cool I am already part of the slack, Right now I think I'll wait for my mentor (Ian Norris) to reply in the fellowship slack. If he's busy I'll put it up in ruby for good slack as well.
